flytekit/types/structured/bigquery.py

```diff
@@ -51,7 +51,14 @@ def _read_from_bq(
     frames = []
     for message in reader.rows().pages:
         frames.append(message.to_dataframe())
-    return pd.concat(frames)
+    if len(frames) > 0:
+        df = pd.concat(frames)
```

Comment:
If the table is empty, the length of `read_session.streams` is 0. so I think we should update the code to below.
Let me know if it doesn't work for you
```suggestion
    frames = []
    if len(read_session.streams) > 0:
        stream = read_session.streams[0]
        reader = client.read_rows(stream.name)
        for message in reader.rows().pages:
            frames.append(message.to_dataframe())

    if len(frames) > 0:
        df = pd.concat(frames)
```

Reply:
@dylanwilder could you take a look at this snippet please? Does this work for you?

Reply:
@wild-endeavor I tested this snippet. Seems like there will be a stream even for empty tables. So this does not work.